packages/components/switch/src/SwitchLabel.tsx

```diff
@@ -20,6 +21,14 @@ export interface LabelProps
   disabled?: boolean
 }
 
-export const Label = ({ className, disabled, ...others }: LabelProps) => {
-  return <RadixLabel className={labelStyles({ className, disabled })} {...others} />
+export const Label = ({ className, disabled, htmlFor, ...others }: LabelProps) => {
+  const { labelId } = useFormFieldState()
+
+  return (
+    <RadixLabel
```

Comment:
Here we should use spark label instead

Reply:
Yes please! I think is better so we keep the label centralized in our codebase 🙏🏻